### ISSUE TITLE
support `-forceSerialized` command line switch in ch

### ISSUE THREAD

**Issue by sethbrenith** [Microsoft/ChakraCore#5558]
As @huaiyudavid recently pointed out, jshost respects `-forceSerialized` (checked in `ScriptEngine::CompileUTF8Core`), but ch seems to have no corresponding code.

**Comment by MSLaguana** [comment]
Is that correct? I checked in a change recently that uses `forceSerialized` in a test: https://github.com/Microsoft/ChakraCore/commit/ad5c19ccb975975740e6041d0c0968f1d1b34eb5#diff-a9b1789717914bc449c3d7444e424c42

**Comment by sethbrenith** [comment]
If those tests are using exclude_jshost, then I doubt we're getting any serialization coverage from them.

**Comment by MSLaguana** [comment]
@boingoing do you know what's going on here? Seth told me offline that while this flag exists in ch, it never seems to actually force the serialization, that code only appeared in jshost.

**Comment by kfarnung** [comment]
@boingoing and I looked into this recently and it doesn't appear to be implemented in ch.  It would be helpful to have.

**Comment by boingoing** [comment]
Yeah, this is the case. Ch.exe has a different flag (-serialied I think) which uses a slightly different, ChakraCore only code path. This is a similar situation to the -UseParserStateCache Ch.exe flag.
